code/go/0chain.net/smartcontract/partitions/partitions.go

```diff
@@ -514,26 +514,65 @@ func (p *Partitions) removeItem(
 	return p.loadLastFromPrev(state)
 }
 
-func (p *Partitions) GetRandomItems(state state.StateContextI, r *rand.Rand, vs interface{}) error {
+func (p *Partitions) GetRandomItems(balances state.StateContextI, r *rand.Rand, vs interface{}, prev, partitionSize int) (int, error) {
```

Comment:
We don't need to pass the `partitionSize`, the `Partitions` has `PartitionSize` field.